via/views/view_video.py

```diff
@@ -38,11 +57,34 @@ def youtube(request, url, **kwargs):
         "api": {
             "transcript": {
                 "doc": "Get the transcript of the current video",
-                "url": request.route_url("api.youtube.transcript", video_id=video_id),
+                "url": request.route_url(
+                    "api.youtube.transcript",
+                    video_id=video_id,
+                    transcript_id=_get_transcript_id(
+                        youtube_service, video_url, via_config
+                    ),
+                ),
                 "method": "GET",
                 "headers": {
                     "Authorization": f"Bearer {ViaSecurityPolicy.encode_jwt(request)}"
                 },
             }
         },
     }
+
+
+def _get_transcript_id(youtube_service, url, via_config):
+    # Try the `via.video.lang` param first
+    if transcript_id := via_config.get("video", {}).get("lang"):
+        return transcript_id
+
+    # Then look up a transcript matching our preferences
+    video = youtube_service.get_video_info(video_url=url)
```

Comment:
We could look this up in the parent, but I chose to keep it separate for now.

We could look to unify the look up with a call like:

```
youtube_service.get_video_info(video_url=url, get_captions=True)
```

Where we could use V3 stuff if `get_captions=False` or the V1 stuff if it's true etc.